requirements.txt

```diff
@@ -1 +1,5 @@
+dj-database-url==0.5.0
 Django~=2.2.2
+gunicorn==19.9.0
```

Comment:
This is the only dependency that is related to configuring a web service run command in Heroku.